doccano_client/models/project.py

```diff
@@ -47,7 +47,7 @@ class Project(BaseModel):
     allow_overlapping: bool = False
     grapheme_mode: bool = False
     use_relation: bool = False
-    tags: List[str] = []
+    tags: List[DictStrAny] = []
```

Comment:
This is the only place w/ `List[Dict[str, Any]]` because when the API returns tags, it includes integer values for the `project` and `id` fields.